leftwm/src/config/mod.rs

```diff
@@ -209,12 +209,12 @@ fn default_terminal<'s>() -> &'s str {
         "guake", // at the bottom because of odd behaviour. guake wants F12 and should really be
                  // started using autostart instead of LeftWM keybind.
     ];
-    for t in terms {
-        if is_program_in_path(t) {
-            return t;
-        }
-    }
-    "termite" //no terminal found in path, default to a good one
+
+    // If no terminal found in path, default to a good one
+    terms
+        .iter()
+        .find(|terminal| is_program_in_path(terminal))
+        .unwrap_or(&"termite")
```

Comment:
Would we want to use the opportunity and replace the orphaned `termite` with something alive?

Reply:
Sounded like a good idea to me, but apparently some AUR folks are trying to keep a somewhat-maintained version around (https://aur.archlinux.org/packages/termite/). Maybe should just move it down list?

Reply:
Nice, didn't know that!
Yeah, moving it down the list makes sense.

Reply:
I reconsidered this, and I given our statement that the list is ordered uncommon -> common, it is arready in a good position and it is actually `alacritty` that seems a bit out of place.
But I quess we can save this discussion for another day.

I'll go ahead and merge this. Thanks @vrmiguel!